test/__snapshots__/parser.test.ts.snap

```diff
@@ -167,6 +177,86 @@ UL. SPOKOJNA 3",
 ]
 `;
 
+exports[`Parser Full Examples test1.mt900 1`] = `
+[
+  Statement {
+    "accountIdentification": "0000004047710139",
+    "closingAvailableBalance": undefined,
+    "closingAvailableBalanceDate": undefined,
+    "closingBalance": undefined,
```

Comment:
Do these statements not include opening and closing balance as well as all other fields that are undefined?

Reply:
They don't include any balance information, they just show the transactions with their references and amounts